tools/debugging/stress_test_transfers.py

```diff
@@ -297,7 +297,9 @@ def kill_restart_and_wait_for_server(
 
     # Wait for the process to completely shutdown, this is necessary because
     # concurrent usage of the database is not allowed.
-    node.process.result.get()
+    exit_code = node.process.result.get()
+    if exit_code != 0:
+        raise Exception(f"Node did not shut down cleanly {node!r}")
```

Comment:
Are we catching this exception somewhere or will this kill the stress test?

Reply:
We want to kill the stress test when anything goes wrong, so this is not caught. I would have done an `exit(1)`, but I was afraid that that might prevent other greenlets from doing proper shutdown.